### ISSUE TITLE
On/Off Works, but nothing else shows up in HomeKit

### ISSUE THREAD

**Issue by jackklink** [tavicu/homebridge-samsung-tizen#281]
Hi, Love the potential of this app, but while the On/Off function is working fine, none of the other keys/inputs show as they should in the Home app. I've added many input app options, yet they show as separate accessories that are inaccessible from the main TV control area and don't even work when accessed on their separate tiles.

Please see the attached screen recording.

This is setup with a Samsung UN43RU7100FXZA 4K TV. And again, I am able to control On/Off just fine, but nothing else--And its not just that I can't control it, none of the options appear in the home app for Up/Down/Left/Right, RW, FF, etc.

Thanks for your help!
Jack

https://user-images.githubusercontent.com/2200283/103248543-d9b0f980-491f-11eb-89e2-f969964b8432.mov

**Comment by jackklink** [comment]
I was able to get INPUTs to work fine after starting fresh, but curious about the rest of the control options?

**Comment by mikemaat** [comment]
I can confirm the same on a UN58TU7000FXZC. On/Off works, but when I try to print the app list to the console using the setting in the config, the list is empty. Also don't see any 'remote' controls.

**Comment by jackklink** [comment]
I figured out that the remote controls are only shown in the remote function of Control Center and not in the Home app. Closing this. The documentation or the setup UI needs to make this more clear.

I was able to print the app l list with my UN43RU7100FXZA, but not with my FRAME TV. This app does not play well with the art mode functions of my Frame TV either. 

![IMG_10903BCC1A30-1](https://user-images.githubusercontent.com/2200283/103422021-3eed3080-4b54-11eb-95de-f06469ad4d5a.jpeg)
